src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java

```diff
@@ -911,6 +911,31 @@ private void saveUserOption(String principalId, String userOptionName, String us
         }
     }
 
+    // This logic is based upon the routing-succeeded check from the AccountsPayableActionBase.route() method.
+    private boolean documentRoutingSucceeded() {
```

Comment:
this function name makes me think it will be looking at a document workflow status, or something along those lines. Should this function be renamed to convey it is just looking for a "successfully routed" message?
Is the comment about AccountsPayableActionBase helpful?

Reply:
I'll rename the method for clarity. I also agree that the comment is likely unnecessary, so I'll remove that too.